coordinator/persistence-test/src/main/java/io/stargate/it/exec/ProcessRunner.java

```diff
@@ -159,7 +159,7 @@ public void onProcessComplete(int exitValue) {
 
     @Override
     public void onProcessFailed(ExecuteException e) {
-      LOG.info("{} process {}/{} failed with exception: {}", kind, generation, node, e);
+      LOG.info("{} process {}/{} failed with exception: {}", kind, generation, node, String.valueOf(e));
```

Comment:
Not sure why but I see these in logs:

```
2025-04-15T16:33:51.0618251Z INFO  [process-runner-stargate-0-0] 2025-04-15 16:33:51,042 ProcessRunner.java:162 - Stargate process 0/0 failed with exception: {}
```

so trying to see if method called is different when last arg is `Throwable` (before change) vs `String` with change.

Reply:
Seems to fix the issue!